src/widgets/toggleRefinement/toggleRefinement.js

```diff
@@ -0,0 +1,168 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
+import cx from 'classnames';
+
+import defaultTemplates from './defaultTemplates.js';
+import ToggleRefinement from '../../components/ToggleRefinement.js';
+import connectToggleRefinement from '../../connectors/toggleRefinement/connectToggleRefinement.js';
```

Comment:
At some point we should flatten the folder to something like `connectors/connectToggleRefinement`

Reply:
Yes – let's take care of that in another pass 👍